packages/date-time-picker/stories/date-time-picker.stories.ts

```diff
@@ -9,182 +9,92 @@ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTA
-        hourSegment: { ...hiddenProperty },
-        minuteSegment: { ...hiddenProperty },
-        secondSegment: { ...hiddenProperty },
-        dayPeriodSegment: { ...hiddenProperty },
-        is12HourClock: { ...hiddenProperty },
-
-        // Inherited
-        _dirParent: { ...hiddenProperty },
-        shadowRoot: { ...hiddenProperty },
-        dir: { ...hiddenProperty },
-        isLTR: { ...hiddenProperty },
-        'allowed-keys': { ...hiddenProperty },
-        allowedKeys: { ...hiddenProperty },
-        autocomplete: { ...hiddenProperty },
-        displayValue: { ...hiddenProperty },
-        focused: { ...hiddenProperty },
-        focusElement: { ...hiddenProperty },
-        grows: { ...hiddenProperty },
-        inputElement: { ...hiddenProperty },
-        label: { ...hiddenProperty },
-        maxlength: { ...hiddenProperty },
-        minlength: { ...hiddenProperty },
-        multiline: { ...hiddenProperty },
-        pattern: { ...hiddenProperty },
-        placeholder: { ...hiddenProperty },
-        renderInput: { ...hiddenProperty },
-        renderMultiline: { ...hiddenProperty },
-        type: { ...hiddenProperty },
-        value: { ...hiddenProperty },
-        _type: { ...hiddenProperty },
-        _value: { ...hiddenProperty },
-        input: { ...hiddenProperty },
-    },
-
-    args: {
-        lang: defaultLocale,
-    },
-
+    title: 'Date∕Time Picker',
+    component: 'sp-date-time-picker',
     parameters: {
-        controls: {
-            // Hide "This story is not configured to handle controls" warning
-            hideNoControlsWarning: true,
-        },
         actions: {
             handles: ['onChange'],
         },
     },
 };
 
-interface StoryArgs {
-    lang?: string;
+const timeGranularities: TimeGranularity[] = ['hour', 'minute', 'second'];
 
+type ComponentArgs = {
     selectedDateTime?: Date;
     timeGranularity?: TimeGranularity;
     quiet?: boolean;
     disabled?: boolean;
     readonly?: boolean;
+    autofocus?: boolean;
     valid?: boolean;
     invalid?: boolean;
+};
 
+type StoryArgs = ComponentArgs & {
     onChange?: (dateTime: Date) => void;
+};
 
+interface SpreadStoryArgs {
     [prop: string]: unknown;
 }
 
-const renderTimeField = (
+const renderDateTimePicker = (
     title: string,
     args: StoryArgs = {},
     content: TemplateResult | typeof nothing = nothing
 ): TemplateResult => {
+    const story = html`
+        <h1>${title}</h1>
+        <hr />
+        <sp-date-time-picker
+            ...=${spreadProps(args as SpreadStoryArgs)}
+            @change=${args.onChange}
+        >
+            ${content}
+        </sp-date-time-picker>
+    `;
+
+    const randomId = String(Math.floor(Math.random() * 99999));
+
+    requestAnimationFrame(() => {
+        const container = document.querySelector(
+            `.story-container-${randomId}`
+        );
+
+        if (container) {
+            render(story, container as HTMLElement);
+        }
+    });
```

Comment:
Why is this approach being leveraged as opposed to rendering the story directly in the returned `html`? If it is required by something specific to these elements, please add a comment for the reasoning.

Reply:
@Westbrook, this solution was suggested by you, I don't remember completely but I think this is necessary because we have more than one example using the same component in the same story, and only the first one was being rendered

Reply:
Well, past Westbrook seems like a smart guy...could we comment that as future Westbrook is looking to continue the trend of less intelligent Westbrooks. 🙈